src/components/Profile/Profile.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+// import Dropdown from 'react-dropdown';
+// import 'react-dropdown/style.css'
+import OnClickTextEdit from './OnClickTextEdit'
+
+
+
+export function ProfilePage() {
```

Comment:
I would recommend exporting a component that is named the same as the folder.  It's just nice to be able to assume that I'm going to be importing a component named "Profile" from the "Profile" folder.